charts/firefly-signer/values.yaml

```diff
@@ -0,0 +1,15 @@
+image:
+  repository: ghcr.io/hyperledger/firefly-signer
+  pullPolicy: IfNotPresent
+  # tag: ""
+
+backend:
+  url: http://besu-node-validator-1.quorum.svc.cluster.local:8545
```

Comment:
so Besu runs in a separate namespace? why?

Reply:
It _used_ to but I pulled back into the default namespace with the FireFly components, this update just got missed.

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/d71b1fcaab0080cd86040e1c33c998b062f7aebe